<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>85</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Point Cloud Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QVTKWidget" name="qvtkWidget"/>
    </item>
    <item>
     <widget class="QVTKWidget" name="qvtkWidget_mesh"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveBinary"/>
    <addaction name="separator"/>
    <addaction name="actionTransform"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuAngle_view">
     <property name="title">
      <string>Angle view</string>
     </property>
     <addaction name="actionMainview"/>
     <addaction name="actionLeftview"/>
     <addaction name="actionTopview"/>
    </widget>
    <addaction name="menuAngle_view"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuView_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionData"/>
    <addaction name="actionProperty"/>
    <addaction name="actionConsole"/>
    <addaction name="actionPointsViewer"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionProjection"/>
    <addaction name="actionResample"/>
    <addaction name="separator"/>
    <addaction name="actionMeshsurface"/>
    <addaction name="actionWireframe"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionStatisticalOutlierRemoval"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuView_2"/>
   <addaction name="menuProcess"/>
   <addaction name="menuFilter"/>
   <addaction name="menuOption"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClear"/>
   <addaction name="actionTransform"/>
   <addaction name="separator"/>
   <addaction name="actionMainview"/>
   <addaction name="actionLeftview"/>
   <addaction name="actionTopview"/>
   <addaction name="separator"/>
   <addaction name="actionProjection"/>
   <addaction name="actionResample"/>
   <addaction name="separator"/>
   <addaction name="actionMeshsurface"/>
   <addaction name="actionWireframe"/>
   <addaction name="separator"/>
   <addaction name="actionRemovePoint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="consoleDock">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>140</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableWidget" name="consoleTable">
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Operation</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Note</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Open.png</normaloff>:/Resources/images/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>open a exsting file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Save.png</normaloff>:/Resources/images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>save the file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionResample">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Cube.png</normaloff>:/Resources/images/Cube.png</iconset>
   </property>
   <property name="text">
    <string>Resample</string>
   </property>
   <property name="toolTip">
    <string>Resample</string>
   </property>
   <property name="statusTip">
    <string>generate a cube point cloud</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/images/help.png</normaloff>:/Resources/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="statusTip">
    <string>show help information</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/images/about.png</normaloff>:/Resources/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>show some information of the software</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionTransform">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Change.png</normaloff>:/Resources/images/Change.png</iconset>
   </property>
   <property name="text">
    <string>File Type Transform</string>
   </property>
   <property name="toolTip">
    <string>File Type Transform</string>
   </property>
   <property name="statusTip">
    <string>change the format of the file</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/images/exit.png</normaloff>:/Resources/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMainview">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Frontview.png</normaloff>:/Resources/images/Frontview.png</iconset>
   </property>
   <property name="text">
    <string>Main view</string>
   </property>
  </action>
  <action name="actionLeftview">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Leftview.png</normaloff>:/Resources/images/Leftview.png</iconset>
   </property>
   <property name="text">
    <string>Left view</string>
   </property>
  </action>
  <action name="actionTopview">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Topview.png</normaloff>:/Resources/images/Topview.png</iconset>
   </property>
   <property name="text">
    <string>Top view</string>
   </property>
  </action>
  <action name="actionData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Manager</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionProperty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Property Manager</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Clear.png</normaloff>:/Resources/images/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMeshsurface">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Mesh.png</normaloff>:/Resources/images/Mesh.png</iconset>
   </property>
   <property name="text">
    <string>Surface</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/Frame.png</normaloff>:/Resources/images/Frame.png</iconset>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveBinary">
   <property name="icon">
    <iconset>
     <normaloff>:/Resources/images/saveBinary.png</normaloff>:/Resources/images/saveBinary.png</iconset>
   </property>
   <property name="text">
    <string>Save as binary</string>
   </property>
   <property name="statusTip">
    <string>Save point cloud as binary file</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionProjection">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/Resources/images/projection.png</normaloff>:/Resources/images/projection.png</iconset>
   </property>
   <property name="text">
    <string>Projection</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStatisticalOutlierRemoval">
   <property name="text">
    <string>Statistical Outlier Removal Filter</string>
   </property>
   <property name="toolTip">
    <string>Statistical Outlier Removal Filter</string>
   </property>
  </action>
  <action name="actionRemovePoint">
   <property name="text">
    <string>Remove Select Point</string>
   </property>
   <property name="toolTip">
    <string>Remove Select Point</string>
   </property>
   <property name="shortcut">
    <string>Meta+R</string>
   </property>
  </action>
  <action name="actionPointsViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mesh Viewer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main_window.qrc"/>
 </resources>
 <connections/>
</ui>
